changelog/buildpacks/_posts/2021-10-21-java-openjdk-17 copy.markdown

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Filename is wrong, it should be: `2021-10-21-java-openjdk-17.md` instead of `2021-10-21-java-openjdk-17 copy.markdown `

Reply:
Now it's good